<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplePlugin</class>
 <widget class="QDockWidget" name="SamplePlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="_btn1">
        <property name="text">
         <string>Run Robot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn0">
        <property name="text">
         <string>button 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="_markerBox">
        <item>
         <property name="text">
          <string>Marker1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marker2a</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marker2b</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marker3</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="_backgroundBox">
        <item>
         <property name="text">
          <string>Color1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Color2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Color3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lines1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Texture1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Texture2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Texture3</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="_trackingType">
        <item>
         <property name="text">
          <string>Three Markers</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Single Marker</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vision</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="_motionSpeed">
        <item>
         <property name="text">
          <string>Slow Marker Motion</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium Marker Motion</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fast Marker Motion</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="_spinBox">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="_label">
        <property name="text">
         <string>Label</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
